huff_cli/src/huffc.rs

```diff
@@ -204,6 +208,60 @@ fn main() {
         file_provider: Arc::new(FileSystemFileProvider {}),
     };
 
+    if cli.label_indices {
+        match compiler.grab_contracts() {
+            Ok(contracts) => {
+                if contracts.len() > 1 {
+                    eprintln!(
+                        "{}",
+                        Paint::red("Multiple contracts found. Please specify a single contract and try again.")
+                    );
+                    std::process::exit(1);
+                }
+
+                if let Some(contract) = contracts.first() {
+                    let m_macro = contract.find_macro_by_name("MAIN").unwrap();
+
+                    // Recurse through the macro and generate bytecode
+                    let bytecode_res: BytecodeRes = Codegen::macro_to_bytecode(
+                        m_macro.clone(),
+                        contract,
+                        &mut vec![m_macro],
+                        0,
+                        &mut Vec::default(),
+                        false,
+                        None,
+                    )
+                    .unwrap();
+
+                    // Format the label indices nicely
+                    let label_indices = bytecode_res
+                        .label_indices
+                        .iter()
+                        .map(|(label, index)| format!("{}: {:#04x}", label, index))
+                        .collect::<Vec<_>>()
+                        .join("\n  ");
+
+                    println!("\n------ JUMP LABEL INDICES ------");
+                    println!("  {}", label_indices);
```

Comment:
With regards to the formatting comment, there is a super lib comfy table that was used in the `huffc test` report by @clabby.
Link to usage in other parts of the codebase here: https://github.com/huff-language/huff-rs/blob/94c34e402a46365fee29863aa08558af178c2b94/huff_tests/src/report.rs#L2

Reply:
oh wow, that library is amazing! I didn't know it existed :)

Reply:
Looking way better:

<img width="403" alt="Screenshot 2023-04-16 at 01 12 22" src="https://user-images.githubusercontent.com/48695862/232257387-6d41f226-dec0-4e72-b97f-72655b79b858.png">